code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -386,6 +385,8 @@
 	grant_abilities(C)
 	return
 
+/datum/species/proc/handle_post_login() //Handles anything not already covered by basic species assignment that requires a client (probably preferences related).
```

Comment:
You may have wanted to pull the preference in during cloning, or sudden species changes in general. You can't do that, though, because you can only guarantee the player preferences match a character when the player is first joining the round, since character preferences *can* be changed mid-round.